WordPress/gutenberg
# Adding more border style options

## What?
Fixes https://github.com/WordPress/gutenberg/issues/42993

## Why?
Currently it is possible to set solid, dashed, dotted in border-style property. Need to add more styles as per ticket request.

## How?
Added more style variations such as double, groove, ridge, inset, outset.

## Testing Instructions

- Go to WP admin dashboard.
- Edit any post or page.
- Add any block which has border style support eg: Pull quote block.
- Check border style option.
- Try each style variation and verify it's working in editor and on frontend.


## Screenshots or screencast 

https://github.com/user-attachments/assets/d289cb98-eed6-4399-b048-75009b9f54f3
